Vagrantfile

```diff
@@ -7,8 +7,9 @@ Vagrant.configure('2') do |config|
   config.ssh.insert_key = false
   config.ssh.private_key_path = '~/.vagrant.d/insecure_private_key'
 
-  config.vm.define 'anxs' do |machine|
```

Comment:
Is this related to the PostgreSQL v10 update?  It looks interesting, but isn't related to this Pull Request, right?